Sources/SwiftBasicFormat/generated/BasicFormat.swift

```diff
@@ -48,13 +48,13 @@ open class BasicFormat: SyntaxRewriter {
   open override func visit(_ node: TokenSyntax) -> TokenSyntax {
     var leadingTrivia = node.leadingTrivia
     var trailingTrivia = node.trailingTrivia
-    if requiresLeadingSpace(node) && leadingTrivia.isEmpty && lastRewrittenToken?.trailingTrivia.isEmpty != false {
+    if requiresLeadingSpace(node) && leadingTrivia.isEmpty && lastRewrittenToken? .trailingTrivia.isEmpty != false {
```

Comment:
😬 This seems like a regression. It’s fine now but it would be nice to fix this in BasicFormat for the future